server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -221,12 +217,12 @@ export function configureWebSocketServices(
             try {
                 await tenantManager.verifyToken(claims.tenantId, token);
             } catch (err) {
-                // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject({
-                    // if we don't understand the error, be lenient and allow retry
-                    code: err?.response?.status ?? 401,
```

Comment:
You're no longer ever checking `err.response`, why not?

Reply:
In TenantManager, I replaced all the direct `Axios.*` calls with `BasicRestWrapper.*` which would only throw `NetworkErrors` rather than an `AxiosError` which was what we expected here